joomla/joomla-cms
# [4.2] Load the extension elements from the model in presets download

### Summary of Changes
The presets download view for the admin items, should load the extension elements from the model.

### Testing Instructions
- Create a new administrator menu with the preset "Components Dashboard"
- Go to the menu items of this new menu, there should be only one with the name "Components"
- Click on it a couple of components to hide  
![image](https://user-images.githubusercontent.com/251072/164988102-f81ddf89-8f12-4e75-adbf-8874b0c1ef91.png)
- Save and close the menu item
- Go mack to the menu manager
- Click on export presets for the new menu  
![image](https://user-images.githubusercontent.com/251072/164988185-e6ccdcee-8a61-47b6-a737-9f5c0ef10c4c.png)

### Actual result BEFORE applying this Pull Request
All Works.

### Expected result AFTER applying this Pull Request
All works.
